justfile

```diff
@@ -335,12 +345,20 @@ get-credentials platform=default_platform:
         ;;
     esac
 
+    token=$(nix run -L .#scripts.get-read-token "projects/796962942582/secrets/ghcr-read-token/versions/latest")
+    sed -i "s/^CONTRAST_GHCR_READ=.*/CONTRAST_GHCR_READ=\"${token}\"/" justfile.env
+
 # Load the kubeconfig from the dev cluster.
 get-credentials-dev:
+    #!/usr/bin/env bash
+    set -euo pipefail
     nix run -L .#scripts.get-credentials "projects/796962942582/secrets/hetzner-ax162-snp-kubeconfig/versions/latest"
     sed -i 's/^default_platform=.*/default_platform="Metal-QEMU-SNP"/' justfile.env
     sed -i 's/^node_installer_target_conf_type=.*/node_installer_target_conf_type="k3s"/' justfile.env
 
+    token=$(nix run -L .#scripts.get-read-token "projects/796962942582/secrets/ghcr-read-token/versions/latest")
+    sed -i "s/^CONTRAST_GHCR_READ=.*/CONTRAST_GHCR_READ=\"${token}\"/" justfile.env
```

Comment:
this looks like the wrong place for it. Suggesting to move it to its own target, and the use that in onboarding.

Reply:
Moved to own target, but not sure if calling it in `just onboard` is worth it: `gcloud` would need to be added to the devshell to be able to test if the person is even logged into gcloud yet, and to the login flow otherwise. I'd rather add a sentence to the onboarding docs.

Reply:
Mentioning in the onboarding guide that you should login to gcloud sounds like a good idea. I don't want to have this in the get-credentials targets as these are called frequently and it will lead to a further slow down.